docs/tutorials/articles/sales_dashboard/step_01/step_01.md

```diff
@@ -0,0 +1,163 @@
+    ```
+=== "Markdown"
+    ```python
+    page = """
+    <|{selected_category}|selector|lov={categories}|on_change=change_category|>
+    """
+    ```
+
+Taipy [visual elements](../../../../refmans/gui/viselements/index.md) take many properties. 
+Note that dynamic properties use a quote and brackets syntax. We use `value="{selected_category}"` 
+to signal to Taipy that `selected_category` should change when the user uses the selector. 
+Likewise, if `categories` changes, the selector will get updated with the new values. 
+
+Here, selector needs an associated string variable which will change when a user selects a value, 
+a list of values (lov) to choose from, and a callback function to call when the value changes. 
+We can define them above:
+
+```python
+data = pd.read_csv("data.csv")
+selected_category = "Furniture"
+categories = list(data["Category"].unique())
+
+def change_category(state):
+    # Do nothing for now, we will implement this later
+    return None
+```
+
+We can now add a chart to display the sales of the top 10 countries for the selected category.
+
+=== "Python"
+    ```python
+        tgb.chart(
+            data="{chart_data}",
+            x="State",
+            y="Sales",
+            type="bar",
+            layout="{layout}",
+        )
+    ```
+=== "Markdown"
+    ```
+    <|{chart_data}|chart|x=State|y=Sales|type=bar|layout={layout}|>
+    ```
+
+Taipy charts have many properties. You can create multiple traces, add styling, change the type of chart, etc.
+
+=== "Python"
+    ```python
+    data = {"x_col": [0, 1, 2], "y_col1": [4, 1, 2], "y_col_2": [3, 1, 2]}
+    with tgb.Page() as page:
+        tgb.chart("{data}", x="x_col", y__1="y_col1", y__2="y_col_2", type__1="bar", color__2="red")
+    ```
+=== "Markdown"
+    ```python
+    data = {"x_col": [0, 1, 2], "y_col_1": [4, 2, 1], "y_col_2":[3, 1, 2]}
+    Gui("<|{data}|chart|x=x_col|y[1]=y_col_1|y[2]=y_col_2|type[1]=bar|color[2]=red|>").run()
+    ```
+
+    
+You can check the syntax for charts [here](../../../../refmans/gui/viselements/generic/chart.md). 
+
+You 
+can also directly embed Plotly charts using the `figure` property as we will do in [Step 3](../step_03/step_03.md).
+
+Here we need to provide a Pandas Dataframe with the data to display, the x and y columns to use, the type of chart,
+and a layout dictionary with additional properties.
+
+```python
+chart_data = (
+    data.groupby("State")["Sales"]
+    .sum()
+    .sort_values(ascending=False)
+    .head(10)
+    .reset_index()
+)
+
+layout = {"yaxis": {"title": "Revenue (USD)"}, "title": "Sales by State"}
+```
+
+Lastly, we can add a table to display the data for the selected category.
+
+=== "Python"
+    ```python
+        tgb.table(data="{data}")
+    ```
+=== "Markdown"
+    ```
+    <|{data}|table|>
+    ```
+
+We can now run the application using:
+  
+```python
+if __name__ == "__main__":
+    Gui(page=page).run(title="Sales", dark_mode=False, debug=True)
```

Comment:
I would remove `dark_mode=False, debug=True` as they just add some unnecessary information that doesn't serve the purpose of a tutorial. 
The simpler, the better.

Reply:
I think it's a good place to introduce these two concepts. By just putting them in the code, the readers know that these properties exist. `debug` seems useless here but we had issues with users complaining that they don't get the full stack trace when an error happens and they did not figure it out themselves

Reply:
All right. It's up to you.